src/API/Options.php

```diff
@@ -64,27 +64,80 @@ public function register_routes() {
 	}
 
 	/**
-	 * Check if a given request has access to manage options.
+	 * Check if a given request has access to get options.
+	 *
+	 * @param  WP_REST_Request $request Full details about the request.
+	 * @return WP_Error|boolean
+	 */
+	public function get_item_permissions_check( $request ) {
+		$params      = explode( ',', $request['options'] );
+		$permissions = $this->get_option_permissions( $request );
+
+		if ( ! is_array( $params ) ) {
```

Comment:
This check (and the one in the update function) should probably be something like:

`if ( ! isset( $request['options'] ) || ! is_array( $params ) ) {`

Otherwise, passing a blank option shows the error message below `Sorry, you cannot manage these options.`.

Reply:
Updated the permission check and also added a default check for `manage_options` for undefined permissions.